bus/routes.go

```diff
@@ -1352,10 +1351,11 @@ func (b *Bus) objectsListHandlerPOST(jc jape.Context) {
 	if jc.Decode(&req) != nil {
 		return
 	}
-	if req.Bucket == "" {
-		req.Bucket = api.DefaultBucketName
+	limit := -1
```

Comment:
Defaulting `offset` and `limit` in a F/U. Looks like this went under the radar for some time.